src/models/Scholarships.ts

```diff
@@ -97,6 +97,7 @@ export interface FilterOptions {
   maxAmount?: number;
   grades?: GradeLevel[];
   majors?: String[];
```

Comment:
Tangential but can you lower-case the `s` here for consistency?
```suggestion
  majors?: string[];
```